tools/clang/test/CodeGenSPIRV/spirv.legal.sbuffer.struct.vulkan1.2.hlsl

```diff
@@ -0,0 +1,33 @@
+// RUN: %dxc -T cs_6_3 -fspv-target-env=vulkan1.2 -E main
+
+// CHECK: %Struct = OpTypeStruct %v3float %_ptr_StorageBuffer_type_StructuredBuffer_uint
+struct Struct
+{
+  float3 foo;
+  StructuredBuffer<uint> buffer;
```

Comment:
Please create another struct and add two `StructuredBuffer<uint>` to the struct.
We want to check whether it correct handles both multiple fields and a single field.